src/erlcloud_sns.erl

```diff
@@ -119,19 +121,19 @@ add_permission(TopicArn, Label, Permissions, Config)
 
 
 
--spec create_platform_endpoint/2 :: (string(), string()) -> string().
+-spec create_platform_endpoint/2 :: (parameter(), parameter()) -> string().
 create_platform_endpoint(PlatformApplicationArn, Token) ->
     create_platform_endpoint(PlatformApplicationArn, Token, "").
 
--spec create_platform_endpoint/3 :: (string(), string(), string()) -> string().
+-spec create_platform_endpoint/3 :: (parameter(), parameter(), parameter()) -> string().
 create_platform_endpoint(PlatformApplicationArn, Token, CustomUserData) ->
     create_platform_endpoint(PlatformApplicationArn, Token, CustomUserData, []).
 
--spec create_platform_endpoint/4 :: (string(), string(), string(), [{sns_endpoint_attribute(), string()}]) -> string().
+-spec create_platform_endpoint/4 :: (parameter(), parameter(), parameter(), [{sns_endpoint_attribute(), parameter()}]) -> string().
 create_platform_endpoint(PlatformApplicationArn, Token, CustomUserData, Attributes) ->
     create_platform_endpoint(PlatformApplicationArn, Token, CustomUserData, Attributes, default_config()).
 
--spec create_platform_endpoint/5 :: (string(), string(), string(), [{sns_endpoint_attribute(), string()}], aws_config()) -> string().
+-spec create_platform_endpoint/5 :: (parameter(), parameter(), parameter(), [{sns_endpoint_attribute(), parameter()}], aws_config()) -> string().
```

Comment:
Line 136 is too long: -spec create_platform_endpoint/5 :: (parameter(), parameter(), parameter(), [{sns_endpoint_attribute(), parameter()}], aws_config()) -> string()..